webpack/ForemanTasks/Components/TasksTable/TasksTableActions.js

```diff
@@ -65,3 +72,48 @@ export const resumeTask = (id, name) => async dispatch => {
     );
   }
 };
+
+export const selectAllRows = results => ({
```

Comment:
IMHO these actions should be part of the general Table actions in foreman core, so we can reuse bulk actions in other tables as well. can you add it to our technical debt please?